lms/envs/common.py

```diff
@@ -1654,13 +1654,15 @@ def _make_locale_paths(settings):
         'source_filenames': [
             'certificates/css/main-ltr.css',
             'css/vendor/font-awesome.css',
+            'css/edraak_certificate.css',
```

Comment:
```suggestion
```

I still don't see a reason why we should have it here. How will the certs look without this file, ideally it should look plane/Okay but without Edraak styles.

I've made a PR in the theme repo to address this issue.